target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -252,8 +253,8 @@ define Device/tplink_tl-cpe210-v3
   TPLINK_HEADER_VERSION := 1
   KERNEL := kernel-bin | append-dtb | lzma | tplink-v1-header
   IMAGES := sysupgrade.bin factory.bin
-  IMAGE/sysupgrade.bin := append-rootfs | tplink-safeloader sysupgrade
+  IMAGE/sysupgrade.bin := append-rootfs | tplink-safeloader sysupgrade | check-size $$$$(IMAGE_SIZE)
   IMAGE/factory.bin := append-rootfs | tplink-safeloader factory
-  SUPPORTED_DEVICES := tplink,tl-cpe-210-v3 tl-cpe-210-v3
+  SUPPORTED_DEVICES += tl-cpe-210-v3
```

Comment:
This is only requires for backwards compatibility with e.g. ar71xx. Since the device does not exist there, I think you can omit the whole SUPPORTED_DEVICES line.